resources/lang/en/admin/users/general.php

```diff
@@ -19,6 +19,8 @@
     'print_assigned'    => 'Print All Assigned',
     'email_assigned'    => 'Email List of All Assigned',
     'user_notified'     => 'User has been emailed a list of their currently assigned items.',
+    'auto_assign_label' => 'User should be skipped during auto assignment commands',
```

Comment:
I think it's prefer this to go in the opposite direction, versus the negative. Default that checkbox to checked, set the db default to 1, and change the language to "Include this user when auto-assigning eligible licenses"